internal/services/hdinsight/hdinsight_hadoop_cluster_resource.go

```diff
@@ -95,6 +95,8 @@ func resourceHDInsightHadoopCluster() *pluginsdk.Resource {
 				},
 			},
 
+			"compute_isolation_properties": SchemaHDInsightsComputeIsolation(),
```

Comment:
properties is implied and redundant?
```suggestion
			"compute_isolation": SchemaHDInsightsComputeIsolation(),
```

Reply:
Will change.